### PR TITLE
Update VMRay pack to 1.1.0 and start adoption process

### PR DESCRIPTION
**Pull request by fseele-vmray**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
This PR updates the VMRay pack to version 1.1.0 which adds a new Playbook, new commands and enhances existing commands (see release notes for details).
It also starts the adoption process to move support from Cortex to VMRay.

- Adds the **Detonate URL - VMRay** playbook
- Adds the **vmray-upload-url** command
- Adds the **vmray-get-summary** command
- Adds the **vmray-get-sample-by-hash** command
- Updates the **vmray-get-sample** command
- Updates the **vmray-get-iocs** command

## Minimum version of Cortex XSOAR
- [X] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by marcellarichmond** [comment]
@mchasepan fyi

**Comment by fseele-vmray** [comment]
@mchasepan Thanks for the review, I reverted the changes in the metadata.
Some questions regarding the validation issues:

> [IN140] - The integration vmray is currently in skipped. Please add working tests and unskip.

What does this mean?

> [BC102] - Possible backwards compatibility break, You've changed the context in the file,please undo. the command is:
> vmray-get-iocs

We only added to the context and did not remove or change any values. Is already considered to break backwards compatibility? Is there any way to get details on which fields it complains about exactly?

> [BC104] - Possible backwards compatibility break, You've changed the name of a command or its arg in the file, please undo, the command was:
> vmray-upload-sample

We removed a parameter which is not necessary anymore. I assumed it would not be a problem to remove it from the yml. What is the best way to do this? Leave it in the yml, mark it as being ignored and just don't use it in the code?

**Comment by mchasepan** [comment]
@Shellyber can you please review Felix's [questions](https://github.com/demisto/content/pull/13683#issuecomment-879275498) above?

**Review by Shellyber — COMMENTED** [review]
Thank you for your contribution! Looks good!
Let's schedule a call, I have some questions regarding some of the changes. Also we would like to have a demo for all the changes and the new playbook.

**Comment by Shellyber** [comment]
@fseele-vmray Regarding your question, I'm checking in. I think that you can remove the argument from the yml file also without it breaking BC.

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 id: 79b5d8a6-2636-480c-8e1c-a3ab2e58ffb5
```
[the remark this answers is not in the document]
Comment (fseele-vmray) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml:1]: Is it possible to change the ID without breaking backwards-compatibility?

File: Packs/VMRay/ReleaseNotes/1_1_0.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+*Note: Support for this pack will be moving to the partner around Oct 15, 2021.*
+
+
+#### Overview
```
[the remark this answers is not in the document]
Comment (fseele-vmray) [reply Packs/VMRay/ReleaseNotes/1_1_0.md]: Done

File: Packs/VMRay/Integrations/VMRay/README.md [hunk 3]
```diff
@@ -1,1418 +1,1085 @@
...
+```json
+{
     "VMRay.Sample": [
         {
-            "SHA1": "69df095557346b3c136db4378afd5ee7a4839dcc", 
-            "Created": "2019-05-27T07:48:11", 
-            "SampleID": 3902285, 
-            "FileName": "KeePass-2.41-Setup.exe", 
-            "FileSize": 3301376, 
+            "SHA1": "69df095557346b3c136db4378afd5ee7a4839dcc",
```
[the remark this answers is not in the document]
Comment (fseele-vmray) [reply Packs/VMRay/Integrations/VMRay/README.md:94]: Yes 👍

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shellyber) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml:1]: No. If the id is the same as the name that's ok and we support it

**Comment by Shellyber** [comment]
@fseele-vmray Let's schedule a demo for all the changes and the new playbook

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml [hunk 4]
```diff
@@ -0,0 +1,652 @@
...
+    timertriggers: []
+    ignoreworker: false
+    taskid: c3457fa4-7b07-41cb-b9e5-f0b523141429
+  "12":
+    id: "12"
+    type: condition
+    task:
+      description: check term
+      version: -1
+      name: Is analysis suspicious or malicious?
```
Comment (evisochek) [inline Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml:339]: Please consider to organize the task for malicious and suspicious In a column and not in parallel so the path will be more clear to the user.

**Comment by evisochek** [comment]
Please add image for the playbook.

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL_README.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+| VMRay.Sample.IOC.File.Hashes.SHA1 | The SHA1 hash of the given file. | unknown |
+| VMRay.ThreatIndicator.AnalysisID | The list of connected analysis IDs. | unknown |
+| VMRay.ThreatIndicator.Category | The category of threat indicators. | unknown |
+| VMRay.ThreatIndicator.Classification | The classifications of threat indicators. | unknown |
+| VMRay.ThreatIndicator.ID | The ID of the threat indicator. | unknown |
+| VMRay.ThreatIndicator.Operation | The operation that caused the indicators. | unknown |
+
+## Playbook Image
+---
+![Detonate URL - VMRay](Insert the link to your image here)
```
Comment (evisochek) [inline Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL_README.md:101]: Please add image

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml [hunk 6]
```diff
@@ -0,0 +1,652 @@
...
+  description: The category of threat indicators.
+- contextPath: VMRay.ThreatIndicator.Classification
+  description: The classifications of threat indicators.
+- contextPath: VMRay.ThreatIndicator.ID
+  description: The ID of the threat indicator.
+- contextPath: VMRay.ThreatIndicator.Operation
+  description: The operation that caused the indicators.
+fromversion: 5.5.0
+tests:
+- no test
```
Comment (evisochek) [inline Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml:652]: I see that there is VMRay-Test-URL playbook, should he test the VMRay-Detonate-URL playbook? if so please add him.

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml (hunk 4 above)
Comment (fseele-vmray) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml:284]: Done in 2b657c1

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL_README.md (hunk 5 above)
Comment (fseele-vmray) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL_README.md]: Done in b775f79 and a8e1965

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (fseele-vmray) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml]: I applied the change but now `demisto-sdk upload` fails with this error:
```
FAILED UPLOADS:
╒══════════════════════════════════╤══════════╤══════════════════════════════════════════════════════════════════════════════════╕
│ NAME                             │ TYPE     │ ERROR                                                                            │
╞══════════════════════════════════╪══════════╪══════════════════════════════════════════════════════════════════════════════════╡
│ playbook-VMRay-Detonate-File.yml │ playbook │ Detonate File - VMRay (79b5d8a6-2636-480c-8e1c-a3ab2e58ffb5) already exists (33) │
╘══════════════════════════════════╧══════════╧══════════════════════════════════════════════════════════════════════════════════╛
```
But this change will not cause any problems for users?

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml (hunk 6 above)
Comment (fseele-vmray) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-URL.yml]: Done in1faa07e1a7c353b5645af0f521e78c209fe3ac92

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (Shellyber) [reply Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml]: @fseele-vmray The playbook already exists in your XSOAR. So it says that the ID already exists. We can't upload two content items with the same id. 
It won't cause problems to users. If you delete the existing playbook in your env, you'll be able to upload this as well.

File: Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
-id: 79b5d8a6-2636-480c-8e1c-a3ab2e58ffb5
+id: Detonate File - VMRay
```
Comment (Shellyber) [inline Packs/VMRay/Playbooks/playbook-VMRay-Detonate-File.yml:1]:
```suggestion
id: 79b5d8a6-2636-480c-8e1c-a3ab2e58ffb5
```

**Review by Shellyber — APPROVED** [review]

**Pull request merged**